app/controllers/editor_controller.rb

```diff
@@ -17,9 +18,24 @@ def post
       params[:body] = node.body if node
     end
     redirect_to "/questions/new?#{request.env['QUERY_STRING']}" if params[:tags] && params[:tags].include?("question:")
+    render template: 'editor/post'
+  end
+
+  def editor
+    redirect_to "/post"
```

Comment:
I think we also are losing the parameters when we redirect -- we should add `?#{request.env['QUERY_STRING']}` to this and any other new redirects we added here -- to preserve the URL parameters, see?